pages/category.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Max Quiz</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://maxui.netlify.app/components/max.css" />
+    <link rel="stylesheet" href="../styles/index.css" />
+    <script src="https://kit.fontawesome.com/8269a6457c.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <a href="/">
+          <div class="nav__logo">
+            <img src="../images/logo.png" alt="logo" width="200" />
+          </div>
+        </a>
+
+        <div class="nav__links">
+          <a href="#categories" class="nav__link">
+            <i class="fa-solid fa-crown"></i>
+            <span class="nav__link--text">Leaderboard</span>
+          </a>
+          <a href="https://github.com.jalajcodes/max-quiz" class="nav__link">
+            <button class="btn btn--primary btn--outline btn--round">
+              <i class="fa-solid fa-user"></i>
+              Account
+            </button>
+          </a>
+        </div>
+      </nav>
+    </header>
+
+    <section class="quizzes container" id="quizzes">
+      <h2>Movie Quizzes</h2>
+      <div class="grid col-3">
+        <div class="card vertical">
+          <div class="card__image">
+            <img
+              class="img-responsive"
+              src="https://thumbs.dreamstime.com/b/male-hand-holding-megaphone-quiz-time-speech-bubble-loudspeaker-banner-business-marketing-advertising-vector-125104939.jpg"
+              alt=""
```

Comment:
suggestion - don't leave alt attribute inside img empty, put some text about the contents of the image